test/test_torch.py

```diff
@@ -16706,6 +16417,216 @@ def caller(cls,
     for test in tensor_op_tests:
         caller(cls, *test)
 
+def _generic_input_fn(dtype, device):
+    input = []
+    input.append(list(range(-5, 5)))
+    input.append([0 for x in range(-5, 5)])
+    input.append([x + 1e-6 for x in range(-5, 5)])
+    # Some vectorized implementations don't support large ranges
```

Comment:
I'm not sure what this comment means here?

Reply:
copy-pasted from original, I think they meant large values rather than large ranges judging by the next line